docs/development/contributing.md

```diff
@@ -46,8 +46,8 @@ associated processes, if you require mediation please leave a mention via
 `@ILIAS-eLearning/technical-board` in the discussion of the PR.
 
 Please make sure to understand that the ILIAS society has a [process for
-functional feature request](https://www.ilias.de/docu/goto_docu_wiki_wpage_788_1357.html) 
-before starting to create your PR. Your PR should thus only contain bug fixes or 
+functional feature request](https://www.ilias.de/docu/goto_docu_wiki_wpage_788_1357.html)
```

Comment:
This URL is outdated. It should be https://docu.ilias.de/goto_docu_wiki_wpage_788_1357.html